doc/contrib/index.rst

```diff
@@ -0,0 +1,101 @@
+.. _contribindex:
+
+Contributing to Arbor
+=====================
+
+First off, thank you for considering contributing to Arbor! It’s people
+like you that make Arbor such a great tool. Feel welcome and read the
```

Comment:
The following is a bit "folksy"
```
First off, thank you for considering contributing to Arbor! It’s people
like you that make Arbor such a great tool.
```
How about:
```
Arbor is an open source community project, and contributions are very welcome. This page discusses the different ways to contribute, from filing bugs and asking questions, to contributing to code and documentation.
```

Reply:
It's loosely based on RTD's own and vscode's contributing doc. Most current style is a bit flat, which I hope will change with time. Here in particular a folksy tone helps be more inviting.

Reply:
I had a read of the VS Code documentation, and I like its tone: it is relaxed and not very formal, but very direct and  focuses on just presenting the information. I don't feel that it has this tone, or the tone I commented on below.

Ironically, RTD's documentation is a single page with very little documentation (what are they trying to tell us?), so they aren't a good example. 

Should we can have a discussion on Slack about what are people's preferences for writing style?

Example from VS Code getting started guide:
https://code.visualstudio.com/api/get-started/your-first-extension

Reply:
Relaxed and concise would be my preference. I'll admit I usually need a few tries to get there (concision), but I can't do it without the intermediate attempts. Not too formal is included in relaxed as far as I am concerned.

Not sure we need to codify all this; we should be clear this is just a guideline, not a requirement for new submissions. We need to be accommodating to diverse backgrounds and levels of command of English.

Some more personal prefs:
* Prefer active voice.
* Use first/third person over passive voice when the agent is clearly identifiable.
* Avoid the imperative style, but not at all cost, and if you do, avoid adding in weasel words (No please, please! I plead guilty too).
* Feel free to write down contractions (the 19th century is over).

For the contrib pages specifically, I want the text to convey some extra enthusiasm.